### ISSUE TITLE
Inplace recovery options?

### ISSUE THREAD

**Issue by DeveeiveD** [jivanpal/drat#46]
This tool is exactly what I was looking for, however I'm curious why some in-place means of recovery are N/A. In my case I've got a drive which I used along with paragon apfs, obviously 3rd party apfs adapter blahh, blahhh.. 

Well so I have a peculiar situation, when I mount ext apfs drive with paragon apfs once on windows and plug it back into osx, drive works. As soon as I replug in osx, it fails with spaceman errors and zeroed-out block error. I'm like uber certain this issues could be fixed by simply relinking blocks (I'm assuming failed writes at some point in the past), however I see that the only solution is decent knowledge of apfs and hex editor.... 

Like I think it would be nice to be able to mod mappings, unless I'm crazy, but saving a snapshot (of few MBs) of block device should be enough for safety and that would be useful in the sense tht one wouldn't have to offload terrabytes of data Xd

**Comment by DeveeiveD** [comment]
Aah, Ic there is commented out modify command in headers. Looking at source yeah I see it's just unfinished afaik

**Comment by jivanpal** [comment]
Yeah, that's right, I want to implement this eventually, it's just quite the task to generalise. I'll leave this issue open to track progress on this feature, thanks for opening it!
